joy2twist/launch/gamepad_controller.launch.py

```diff
@@ -0,0 +1,39 @@
+import os
+from ament_index_python.packages import get_package_share_directory
+from launch import LaunchDescription
+from launch.actions import IncludeLaunchDescription, ExecuteProcess, DeclareLaunchArgument
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import LaunchConfiguration
+
+
+def generate_launch_description():
+    ld = LaunchDescription()
+
+    joy2twist_cfg_path = os.path.join(get_package_share_directory(
+        'joy2twist'), 'config', 'joy2twist.yaml')
+
+    declare_joy2twist_params_file_argument = DeclareLaunchArgument(
+        'joy2twist_params_file',
+        default_value=joy2twist_cfg_path,
+        description='ROS2 parameters file to use with joy2twist node')
+    ld.add_action(declare_joy2twist_params_file_argument)
+
+    run_joy_linux = ExecuteProcess(
```

Comment:
Why is it executed as process and not as node?

Reply:
Słuszna uwaga

Reply:
:heavy_check_mark: